vuepress/docs/next/docs/reference/databases.md

```diff
@@ -347,3 +347,13 @@ After updating parameters with the one you need, you should end with a yaml like
 Using `spec.dbms: "none"` will cause the operator to skip that initial schema/user creation step entirely.
 Then adding those variables under the `spec.environmentVariables` section will supply connection parameters that will be used by EntandoApp.
 Keep in mind that all these parameters will be applied to each of the containers in the EntandoApp pod and that they will also override existing values.
+
+## Liquibase Migration
+
+Beginning with Entando 7.0, AppEngine (DeApp) modules will implement automatic Liquibase migrations to manage structural changes to databases running on MySQL or PostgreSQL.
+
+The parameter provided to the environment variable `DB_MIGRATION_STRATEGY` determines how required updates are applied to components of an existing database. Three database migration modes are supported and govern upgrade behavior:
+
+- `auto` (default setting): The application will start and the database will update. Changes will be applied to each component introduced in Entando 7.0 and beyond.
```

Comment:
can the 'will' be taken out of the verb clauses, so it's more direct. "The application starts and the database updates." & "Changes are applied to each component..." Same for the following 2 options. 

"and beyond" reminds me of Toy Story:)

Reply:
yes - and that matches with our style guide instruction to use the present tense when describing a process, even if it has yet to be followed :) 

lol i'd have to really dig to figure out if vuepress supports the "greater than or equal to" symbol and the markdown syntax...